### PR TITLE
Add options to badges

### PR DESCRIPTION
**Pull request by iantrich**
https://github.com/home-assistant/home-assistant-polymer/pull/3552

**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10286"></a>

### CODE REVIEW

**Review by balloob — APPROVED** [review]

**Review by balloob — DISMISSED** [review]
ok to merge when the syntax error is fixed.

**Comment by iantrich** [comment]
@balloob I'm not seeing the error?

**Comment by balloob** [comment]
Neither do I, but netlify says

```
8:51:37 AM:   Liquid Exception: undefined method `split' for nil:NilClass in lovelace/views.markdown
8:51:37 AM: jekyll 3.8.6 | Error:  undefined method `split' for nil:NilClass
8:51:37 AM: Generating site failed
```

**Comment by iantrich** [comment]
I have no idea what that means 😂

**Comment by iantrich** [comment]
@frenck any ideas by chance?

File: source/lovelace/views.markdown [hunk 1]
```diff
@@ -52,6 +52,44 @@ views:
...
+{% endconfiguration %}
+
+### Example
+
+View config:
+
+```yaml
+- title: Living room
+  badges:
+    - device_tracker.demo_paulus
```
Comment (fabaff) [inline source/lovelace/views.markdown:85]:
```suggestion
    - entity: device_tracker.demo_paulus
```

Comment (iantrich) [reply source/lovelace/views.markdown:85]: the `entity` prefix is only required if you wish to set other options for the entity

**Comment by frenck** [comment]
Hmm, I don't see a direct reason on why this fails. Will assign me and pick to one up later today. Interested to find out why this fails.

**Comment by frenck** [comment]
@iantrich Found it! Although, this is more a workaround. Our configuration plugin assumes an integration/component/platform page, which in this case, it isn't. Providing an additional (meaningless) argument to the `configuration` tag fixes it for now (as it has been done before as it seems).

Need to revisit the part of the code. For now, this build should pass.

**Review by frenck — APPROVED** [review]
🎉 Merging

**Pull request merged**